src/ontology/edits/oeo-physical.omn

```diff
@@ -6316,6 +6316,52 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/817",
 Class: OEO_00240003
 
     
+Class: OEO_00240009
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Combined heat and power generation (CHP) is an energy transformation that converts energy simultaneously to electrical energy and thermal energy."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "co-generation
+CHP"@en,
```

Comment:
These should be two alternative term annotations, one for _co-generation_ and one for _CHP_.

Reply:
Sorry, my fault. Just fixed it.